lib/mayaUsd/base/tokens.h

```diff
@@ -28,9 +28,17 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 // Tokens that are used as optionVars in MayaUSD
 //
-#define MAYA_USD_OPTIONVAR_TOKENS                        \
-    /* Always target a session layer on a mayaUsdProxy*/ \
-    (mayaUsd_ProxyTargetsSessionLayerOnOpen)
+// mayaUsd_ProxyTargetsSessionLayerOnOpen: Always target a session layer on a mayaUsdProxy
+// mayaUsd_SaveLayerFormatArgBinaryOption: When saving as .usd, should the format be binary
+// mayaUsd_SerializedUsdEditsLocation: Option for what to do with Usd edits when the current
+// Maya scene is about to be saved.  optionVar values are:
+//    1: save all edits back to usd files.
+//    2: export the dirty usd layers to Maya string attributes to be serialized to the Maya file.
+//    3: ignore all Usd edits.
+// mayaUsd_SerializedUsdEditsLocationPrompt: optionVar to force a prompt on every save
+#define MAYA_USD_OPTIONVAR_TOKENS                                                     \
+    (mayaUsd_ProxyTargetsSessionLayerOnOpen)(mayaUsd_SaveLayerFormatArgBinaryOption)( \
```

Comment:
Minor, but I'd like to stick the the convention of using `// clang-format off` for token defintions - ie, like in [jobArgs.h](https://github.com/Autodesk/maya-usd/blob/1ac15c974d7f57f5e0c524508779cad5abbcdc15/lib/mayaUsd/fileio/jobs/jobArgs.h#L37).  I know, I was one of the big proponents of `clang-format`, but I think this is an exception where it looks much nicer manually formatted, and makes for much cleaner diffs if we need to add/remove tokens later...

Reply:
Yes!  Sean actually mentioned that to me when I complained that it kept formatting the change like this, but I forgot to go back and add that.

Reply:
Just wanted to flag also that there was a bit of clean-up in this area from #1119 that probably created a merge conflict that needs resolving. I put the `mayaUsd_` prefix in the values only:

```C++
...
    ((SaveLayerFormatArgBinaryOption, "mayaUsd_SaveLayerFormatArgBinaryOption")) \
...
```